Extensions/ArtifactEngine/Engine/artifactEngine.ts

```diff
@@ -33,11 +33,11 @@ export class ArtifactEngine {
                     this.logger.logSummary();
                     resolve(this.artifactItemStore.getTickets());
                 }, (err) => {
-                    ci.publishEvent('issue', { issueType: 'error', errorMessage: err });
+                    ci.publishEvent('reliability', <ci.IReliabilityData>{ issueType: 'error', errorMessage: err });
                     reject(err);
                 });
             }, (err) => {
-                ci.publishEvent('issue', { issueType: 'error', errorMessage: err });
+                ci.publishEvent('reliability', { issueType: 'error', errorMessage: err });
```

Comment:
no need to type case to <ci.IReliabilityData> here?

Reply:
not strictly required but better to cast.